doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst

```diff
@@ -65,23 +65,43 @@ There are two implementations currently supported:
   .. figure:: images/bootloader_memory_layout.svg
      :alt: Memory layout
 
-For detailed information about the memory layout, see the partition configuration in the :file:`partitions.yml` file, located in the build folder directory, or run ``ninja partition_manager_report``.
+By default, building an application with any bootloader configuration creates a :ref:`multi-image build <ug_multi_image>`, where the :ref:`partition_manager` manages its memory partitions.
+Bootloaders are built as child images.
```

Comment:
If we are to be very correct, the sentence `Bootloaders are built as child images.` as a standalone is not technically true.
However, if we see it in the contect of the preceding sentence, I think it is fine.
Maybe refer to make sentences more connected?
`In this case, bootloaders are built as child images.`